fiber-grpc/go.sum

```diff
@@ -1,1047 +1,54 @@
-cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Why did you delete go.sum

Reply:
go.sum is here maybe something is wrong 
[link](https://github.com/muratmirgun/recipes/tree/master/fiber-grpc)

Reply:
go.mod version 1.15 -> 1.19 some unused go.sum lines deleted only. go.mod is not fully deleted

Reply:
Oh sorry 🤦

Reply:
Thanks Have a nice weekend ✋